azure-pipelines/experiment/msal-code-coverage-experiment-clone.yml

```diff
@@ -0,0 +1,59 @@
+# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
+#       env:
+#           MY_ACCESS_TOKEN: $(System.AccessToken)
+# Variable 'android:serverUrl' was defined in the Variables tab
+# Variable 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable 'mvnAccessToken' was defined in the Variables tab
+# Variable 'test_repo_branch' was defined in the Variables tab
+# Variable 'test_repo_dir' was defined in the Variables tab
+# Variable 'testEmailAccount' was defined in the Variables tab
+# Variable 'testEmailAccountPassword' was defined in the Variables tab
+# Variable 'toEmailAccount' was defined in the Variables tab
+name: $(date:yyyyMMdd)$(rev:.r)
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: aymeghwa-CodeCoverage
+  - repository: common
+    type: github
+    name: AzureAD/microsoft-authentication-library-common-for-android
```

Comment:
weird....is this a pipeline for the common repo? Seems like it is, then why is it being checked into MSAL repo?

Reply:
Actually seems it is meant to be for MSAL....but the repository block seems wrong

Reply:
The pipeline is for the msal repo, but it uses templates that exists in the common repo

Reply:
MSAL has a submodule for common; however I think in this case you want the latest files in common is that correct?